packs/data/pathfinder-bestiary-2.db/filth-fire.json

```diff
@@ -307,7 +307,7 @@
                     "value": 2
                 },
                 "description": {
-                    "value": "<p>Toxic materials and churning rubbish within the filth fire's body explode in one of three ways. The filth fire chooses the effect, but it can't make the same choice twice in a row.</p><ul><li>Fiery Beam (evocation, fire, primal) The filth fire expels a @Template[type:line|distance:30] of flame that deals [[/r {3d6}[fire]]]{3d6 fire damage} with a @Check[type:reflex|dc:21|basic:true] save.</li><li>Shrapnel Blast (evocation, primal) The filth fire shoots jagged rubbish out in a @Template[type:emanation|distance:5] that deals [[/r {2d12}[piercing]]]{2d12 piercing damage} with a @Check[type:reflex|dc:21|basic:true] save.</li><li>Toxic Fumes (evocation, poison, primal) The filth fire belches a @Template[type:cone|distance:15] of toxic smoke that deals [[/r {2d6}[poison]]]{2d6 poison damage} (@Check[type:fortitude|dc:21|basic:true] save). A creature that fails is also @Compendium[pf2e.conditionitems.Sickened]{Sickened 1} (or sickened 2 on a critical failure).</li></ul>"
+                    "value": "<p>Toxic materials and churning rubbish within the filth fire's body explode in one of three ways. The filth fire chooses the effect, but it can't make the same choice twice in a row.</p>\n<ul>\n<li><strong>Fiery Beam</strong> (evocation, fire, primal) The filth fire expels a @Template[type:line|distance:30] of flame that deals [[/r {3d6}[fire]]]{3d6 fire damage} with a @Check[type:reflex|dc:21|basic:true|traits:evocation,fire,primal] save.</li>\n<li><strong>Shrapnel Blast</strong> (evocation, primal) The filth fire shoots jagged rubbish out in a @Template[type:emanation|distance:5] that deals [[/r {2d12}[piercing]]]{2d12 piercing damage} with a @Check[type:reflex|dc:21|basic:true|traits:evocation,primal] save.</li>\n<li><strong>Toxic Fumes</strong> (evocation, poison, primal) The filth fire belches a @Template[type:cone|distance:15] of toxic smoke that deals [[/r {2d6}[poison]]]{2d6 poison damage} (@Check[type:fortitude|dc:21|basic:true|traits:evocation,poison,primal] save). A creature that fails is also @Compendium[pf2e.conditionitems.Sickened]{Sickened 1} (or @Compendium[pf2e.conditionitems.Sickened]{Sickened 2} on a critical failure).</li>\n</ul>"
```

Comment:
```suggestion
                    "value": "<p>Toxic materials and churning rubbish within the filth fire's body explode in one of three ways. The filth fire chooses the effect, but it can't make the same choice twice in a row.</p>\n<ul>\n<li><strong>Fiery Beam</strong> (evocation, fire, primal) The filth fire expels a @Template[type:line|distance:30] of flame that deals [[/r {3d6}[fire]]]{3d6 fire damage} with a @Check[type:reflex|dc:21|basic:true|traits:evocation,fire,primal|overrideTraits:true] save.</li>\n<li><strong>Shrapnel Blast</strong> (evocation, primal) The filth fire shoots jagged rubbish out in a @Template[type:emanation|distance:5] that deals [[/r {2d12}[piercing]]]{2d12 piercing damage} with a @Check[type:reflex|dc:21|basic:true|traits:evocation,primal|overrideTraits:true] save.</li>\n<li><strong>Toxic Fumes</strong> (evocation, poison, primal) The filth fire belches a @Template[type:cone|distance:15] of toxic smoke that deals [[/r {2d6}[poison]]]{2d6 poison damage} (@Check[type:fortitude|dc:21|basic:true|traits:evocation,poison,primal|overrideTraits:true] save). A creature that fails is also @Compendium[pf2e.conditionitems.Sickened]{Sickened 1} (or @Compendium[pf2e.conditionitems.Sickened]{Sickened 2} on a critical failure).</li>\n</ul>"
```